Snacc-Overflow/student-toolbox
# Grade Calculator Course Menu Accordion

# Grade Calculator Course Menu Accordion

## Related Issue
Closes #58 

## Description

- When clicking on a course to show its menu/list of assignments, the menu will be expanded and collapsed using an accordion design.

## Testing

- Click on course to expand. Menu should expand with a transition and should have a satisfactory feel to it.
- Click on course to collapse. Similar to above.
- All previous functionality working as intended
